### PR TITLE
Alternative

### PR DESCRIPTION
**Pull request by Sisanu**
Maybe some helper steps
https://www.loom.com/share/240e7c697d9642d299068520174fe5f9

### CODE REVIEW

**Comment by zpaceinvader** [comment]
Could you combine your instructions with the existing instructions for local symlink method? or replace it?

File: INSTALLATION.md [hunk 1]
```diff
@@ -82,3 +82,47 @@ Bedrock documentation is available at [https://roots.io/bedrock/docs/](https://r
...
+
+`$ ln -s $PATH/sample1/app/sample1/public/content/ $PATH/sample1/app/public/wp-content`
+
+If that worked, check the site. 
+
+**IMPORTANT!** The theme is not activated, you will see the WSOD.
+Login to admin and activate the theme.
+
+**NOTE!**
+The theme that comes with the project setup is built, and cannot be used for development.
```
Comment (zpaceinvader) [inline INSTALLATION.md:125]: teft-theme doesn't come with the project any more

File: INSTALLATION.md [hunk 2]
```diff
@@ -82,3 +82,47 @@ Bedrock documentation is available at [https://roots.io/bedrock/docs/](https://r
...
+$ `composer create-project dekode/project-base sample1`
+
+At this point you should have a structure like:
+- app 
+	- public
+	- sample1
+- conf 
+- logs
+
+If at this step you get any errors, check the constrains,
```
Comment (zpaceinvader) [inline INSTALLATION.md:107]: could you update the current constraints to something that is working?

File: INSTALLATION.md (hunk 1 above)
Comment (Sisanu) [reply INSTALLATION.md:125]: I must be crazy or something, look at the screen recording I did like 30 minutes ago.

File: INSTALLATION.md (hunk 2 above)
Comment (Sisanu) [reply INSTALLATION.md:107]: What do you mean? Is that not something visible in the video?

Comment (zpaceinvader) [reply INSTALLATION.md:107]: No just update the composer.json in this repo to something valid

File: INSTALLATION.md (hunk 1 above)
Comment (zpaceinvader) [reply INSTALLATION.md:125]: Oh, it was removed by stian yesterday. not sure what's going on there

File: INSTALLATION.md (hunk 2 above)
Comment (Sisanu) [reply INSTALLATION.md:107]: This will break anyway for other dependencies too. Maybe you can change it, I don't want to touch this project code for now.

**Pull request closed without merging**